src/clj/qbits/jet/client/http.clj

```diff
@@ -130,17 +132,16 @@
   (encode-chunk [x]
     (ByteBuffer/wrap x))
   (encode-body [ba]
-    (BytesContentProvider.
-      (into-array array-class [ba])))
+    (BytesContentProvider. nil (into-array array-class [ba])))
```

Comment:
Why did we add `nil`s to the providers?

Reply:
Some of these content providers implement a typed interface that ends up assigning a default `content-type` header when one is not provided in the request. Setting the value to `nil` explicitly avoids this behavior.